File: 0x12-javascript-warm_up/2-arguments.js
#!/usr/bin/node
// prints a message based on the number of arguments
let i;
for (i in process.argv) {
}
i--;
if (i === 0) {
  console.log('No argument');
}
if (i === 1) {
  console.log('Argument found');
}
if (i > 1) {
  console.log('Arguments found');
}

File: 0x12-javascript-warm_up/Rusty/2-arguments.rs
/* prints a message based on the number of arguments */
use std::env;

fn main() {
	let arg_len = env::args().count() - 1;

	if arg_len == 0 {
		println!("No argument");
	}
	else if arg_len == 1 {
		println!("Argument found");
	}
	else if arg_len > 1 {
		println!("Arguments found");
	}
}
